### PR TITLE
[W5.11][F09-4]Rachel Lee

### PR DESCRIPTION
**Pull request by leerachel**

### CODE REVIEW

**Comment by MingXunSeah** [comment]
Good use of variable names that made the code a lot more readable.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -5,26 +5,39 @@
...
-        this.keywords = keywords;
+    public FindCommand(Set<String> keywords) { this.keywords = keywords; }
+
+    /**
+     * Returns a copy of keywords, in small case, in this command.
+     */
+    public Set<String> ToSmallCaseKeywords(Set<String> keywords) {
+        //Converting set to array to iterate through
+        String[] keywordsArray = keywords.toArray(new String[keywords.size()]);
+        for (int i = 0; i < keywordsArray.length; i++) {
```
Comment (sxqnicholas96) [inline src/seedu/addressbook/commands/FindCommand.java]: Nicely commented code, making it very readable and clear. A tip: You can use enhanced iterators to iterate thru the set here!

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -28,12 +29,23 @@ public FindCommand(Set<String> keywords) {
...
-    public Set<String> getKeywords() {
-        return new HashSet<>(keywords);
+    public Set<String> ToSmallCaseKeywords(Set<String> keywords) {
+        //Converting set to array to iterate through
+        String[] keywordsArray = keywords.toArray(new String[keywords.size()]);
+        for (int i = 0; i < keywordsArray.length; i++) {
+            keywordsArray[i] = keywordsArray[i].toLowerCase();
+        }
+        Set<String> keywordsInSmallCase = new HashSet<>(Arrays.asList(keywordsArray));
+        return keywordsInSmallCase;
```
Comment (fanyiii) [inline src/seedu/addressbook/commands/FindCommand.java:41]: Good job implementing this additional method to convert the keywords to lower case. Remember to follow the naming convention for method though! (should be toSmallCaseKeywords instead)

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -47,10 +59,12 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
```
Comment (fanyiii) [inline src/seedu/addressbook/commands/FindCommand.java:61]: Nice new implementation!

File: test/expected.txt [hunk 4]
```diff
@@ -213,8 +213,9 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
-|| 
-|| 0 persons listed!
+|| 	1. Betsy Choo Tags: [secretive]
+||
+|| 1 persons listed!
```
Comment (fanyiii) [inline test/expected.txt:218]: You can try coming up with more test cases, e.g. keywords in upper case etc.

**Review by fanyiii — COMMENTED** [review]
Good job implementing the functionality and updating the tests&user guide. Next time try to create a cleaner PR (without the changes in irrelevant folder). You may close this PR after reviewing

**Pull request closed without merging**